### PR TITLE
Fix news in the news recommendations widget

### PR DESCRIPTION
**Pull request by yuriizhurakovskyi**

### CODE REVIEW

File: core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java [hunk 1]
```diff
@@ -119,9 +121,46 @@ public AddEcoNewsDtoResponse save(AddEcoNewsDtoRequest addEcoNewsDtoRequest,
...
+
+    /**
+     * {@inheritDoc}
+     *
+     * @author Zhurakovskyi Yurii.
+     */
+    @Override
+    public List<EcoNewsDto> getThreeRecommendedEcoNewsByTags(List<String> tags, Long openedEcoNewsId) {
+        List<EcoNews> ecoNewsList = ecoNewsRepo.getThreeRecommendedEcoNewsByTags(tags, openedEcoNewsId);
+        if (ecoNewsList.size() < 3) {
```
Comment (antonbakalets) [inline core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:154]: Magic Number Anti-pattern.

File: core/src/main/java/greencity/controller/EcoNewsController.java [hunk 2]
```diff
@@ -165,4 +165,30 @@ public EcoNewsController(EcoNewsService ecoNewsService) {
...
+            @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+    })
+    @GetMapping("/recommended")
+    public ResponseEntity<List<EcoNewsDto>> getThreeRecommendedEcoNews(
+            @ApiParam(value = "Tags for priorities(if do not input tags get without priorities)")
+            @RequestParam(required = false) List<String> tags,
+            @RequestParam(required = true) Long openedEcoNewsId
+    ) {
+        if (tags == null || tags.isEmpty()) {
```
Comment (antonbakalets) [inline core/src/main/java/greencity/controller/EcoNewsController.java:187]: - I would move this check to service level
- I would add 'else' to explicitly indicate that those are two exclusive branches of method execution

File: core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java (hunk 1 above)
Comment (yuriizhurakovskyi) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java]: Replaced by a NUMBER_OF_RECOMMENDED_ECO_NEWS constant

File: core/src/main/java/greencity/controller/EcoNewsController.java (hunk 2 above)
Comment (yuriizhurakovskyi) [reply core/src/main/java/greencity/controller/EcoNewsController.java]: I combined getThreeRecommendedEcoNews, getThreeRecommendedEcoNewsByTags service methods into a single getThreeRecommendedEcoNews service method, moved tag count check to service level

**Pull request merged**